src/berry_mill/kiwrap.py

```diff
@@ -146,6 +146,20 @@ def _check_repokey(self, repodata:Dict[str, str], reponame) -> None:
                                                 path=os.path.join(self._trusted_gpg_d, selected),
                                                 params=None, query=None, fragment=None,
                                                 netloc=None).geturl()
+                    keybox = '/var/cache/kiwi/apt-get/trusted-keybox.gpg'
+                    gpg_args = [
+                            'gpg', '--no-options', '--no-default-keyring',
+                            '--no-auto-check-trustdb', '--trust-model', 'always',
+                            '--keyring', keybox
+                        ]
+                    key = '/etc/apt/trusted.gpg.d/' + selected
```

Comment:
This is:
a) Debian-only. On other systems this will just fail.
b) Doesn't belong here but to a [specific repo implementation handling](https://github.com/isbm/berrymill/blob/main/src/berry_mill/localrepos.py#L91).
c) It should be `$HOME/.gnupg` instead.
d) Why string concatenation instead of `os.path.join`?